### ISSUE TITLE
Print area, tags priority and story points on a card

### ISSUE THREAD

**Issue by ghost** [ryanjones/pcards#10]
Ryan - It would be awesome if the area, tags priority and story points would be shown on the card . A very welcome feature!

Furthermore, I'm not sure what the intent of the boxes Dev and UX is, and how I can utilise them? Also I'm not sure what the intent of the top right square box is and how the text below Product Owner can be set.

**Comment by jeffpriz** [comment]
yeah, I think we would really love to have those values (especially Story Points) printed to the card.. Judging by the fact that this issue is almost a year old with no reply, and that there are open PRs that have no response, I am going to guess that this extension is dead?  maybe I will fork and create a new version to add additional features

**Comment by jeffpriz** [comment]
@highstone Not sure if this is still an issue for you or if you're still interested in printing cards .. But I've forked this and published a new extension name "Print Physical Card"  https://marketplace.visualstudio.com/items?itemName=OneLuckiDev.ppcards

I think you'll find it addresses things you're looking for

**Comment by ghost** [comment]
Awesome @jeffpriz, I will definitely take a look!
